internal/services/backoff.go

```diff
@@ -0,0 +1,74 @@
+//
+// Copyright Strimzi authors.
+// License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+//
+
+// Package services defines an interface for canary services and related implementations
+package services
+
+import (
+	"fmt"
+	"time"
+)
+
+const (
+	ScaleDefault       = 200
+	BaseDefault        = 2
```

Comment:
We only really need to support a base of 2, and it would simplify the code quite a bit if we made that assumption.